plugins/synaptics-cape/README.md

```diff
@@ -34,7 +34,13 @@ The vendor ID is set from the USB vendor, in this instance set to `USB:0x1395`
 
 ## Quirk Use
 
-This plugin uses no plugin-specific quirks.
+This plugin uses the following plugin-specific quirks:
+
+### InReportViaInterrupt
+
+If firmware can reply command over interrupt IN EP.
+
+Since: 1.6.5
```

Comment:
I guess we want to backport this to 1.6.x that means?  Or did you not remember to update this to 1.7.0?

Reply:
The github show the latest version number is 1.6.4 released few hours ago. Therefore, I update this to 1.6.5.  Will update it to 1.7.0.

Reply:
OK thanks.
Yeah we should only set it to 1.6.x if this plugin will backport to 1_6_x, which I don't think it will.